### PR TITLE
Sql-math-function-floor

### LINKED ISSUE #2218: [Term Entry] SQL Math Functions FLOOR
### Reason/inspiration (optional)

We would like a new term entry for the Math Function, `FLOOR`, in SQL.

The entry should go in a file in docs/content/sql/concepts/math-functions/terms/floor/floor.md

The entry should: 

- Explain what the math function `FLOOR` is in SQL and what it’s used for, starting with an introductory paragraph (2-3 sentences).
- Include SQL database compatibility as a note.
- Links out to entries as needed.

Please refer to the [entry templates (term entry template in particular)](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

Thanks for assigning me. This closes #2218 . I used the example i removed from the math-functions concept entry. 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/floor/floor.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The SQL math function `FLOOR()` returns the largest integer that is less than or equal to a specified numeric value. It effectively rounds down a number to the nearest whole integer.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/floor/floor.md:18]:
```suggestion
The SQL math function **`FLOOR()`** returns the largest integer that is less than or equal to a specified numeric value. It effectively rounds down a number to the nearest whole integer.
```

File: content/sql/concepts/math-functions/terms/floor/floor.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+In this example, the following data is given, in a table named `employees`:
+
+| employee_id | salary  |
+| ----------- | ------- |
+| 1           | 3500.27 |
+| 2           | 1530.90 |
+| 3           | 3025.10 |
+| 4           | 5040.50 |
+
+The `FLOOR()` function can be used to calculate the largest integer that is less than or equal to a given value:
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/floor/floor.md:38]:
```suggestion
The `FLOOR()` function can be used to return the largest integer that is less than or equal to a given value:
```

File: content/sql/concepts/math-functions/terms/floor/floor.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+```sql
+SELECT employee_id, FLOOR(salary) AS floored_salary
+FROM employees;
+```
+
+In this SQL statement, the `salary` values are floored for each employee to calculate the `floored_salary`. The [`AS`](https://www.codecademy.com/resources/docs/sql/commands/as) keyword is used to rename the `FLOOR(salary)` column to `floored_salary` in the output.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/floor/floor.md:45]:
```suggestion
In this SQL statement, the `salary` values are floored for each employee to get the `floored_salary`. The [`AS`](https://www.codecademy.com/resources/docs/sql/commands/as) keyword is used to name the resulting column `floored_salary` in the output.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj Just a few comments. Tag me when you've finished making the edits.

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj Looks good

File: content/sql/concepts/math-functions/terms/floor/floor.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+The SQL math function **`FLOOR()`** returns the largest integer that is less than or equal to a specified numeric value. It effectively rounds down a number to the nearest whole integer.
+
+## Syntax
+
+```sql
+SELECT FLOOR(number)
+FROM table_name;
+```
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/floor/floor.md:25]: Like to see a sentence explanation of the syntax describing what `number` is.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikolaspoczekaj  I did a review and only had one comment.

**Comment by nikolaspoczekaj** [comment]
Done. @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@nikolaspoczekaj Looks good to merge!

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/sql/math-functions/floor

image

**Pull request merged**